### PR TITLE
Add support for ACL policies

### PR DESCRIPTION
**Pull request by mleneveut**
Fixes: #476

### CODE REVIEW

**Comment by mleneveut** [comment]
Needs hvac >= 2.1.0

File: ansible/modules/hashivault/hashivault_write.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def hashivault_write(module):
                     returned_data = client.secrets.kv.v2.create_or_update_secret(mount_point=mount_point, cas=cas,
                                                                                  path=secret, secret=write_data)
                 else:
-                    returned_data = client.write(secret_path, **write_data)
+                    returned_data = client.write_data(secret_path, **write_data)
```
Comment (TerryHowe) [inline ansible/modules/hashivault/hashivault_write.py:136]: Was this required for hvac 2?

Comment (mleneveut) [reply ansible/modules/hashivault/hashivault_write.py:136]: Yes it's a breaking change of 2.0.0 : https://github.com/hvac/hvac/blob/main/CHANGELOG.md#200

**Review by TerryHowe — COMMENTED** [review]
merged that, might fix this

**Comment by TerryHowe** [comment]
Looks like get bogus policy is failing now for some reason

**Comment by mleneveut** [comment]
@TerryHowe I fixed the test, as hvac returns an InvalidPath exception if the ACL policy doesn't exist : 

```
    try:
        policy = client.sys.read_acl_policy(name)
        policy = policy.get('data', policy).get('policy', policy)
    except InvalidPath as e:
        policy = None
```

**Comment by TerryHowe** [comment]
This and jwt modules release in 5.3.0

**Pull request merged**